account_bank_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -107,6 +108,10 @@ class OnlineBankStatementProvider(models.Model):
     certificate_private_key = fields.Text()
     certificate_chain = fields.Text()
     allow_empty_statements = fields.Boolean()
+    adjust_next_run = fields.Boolean(
```

Comment:
It's safe to have this always enabled, don't think there's need for an option tbh

Reply:
I will leave out the boolean